sdk/python/tests/test_historical_retrieval.py

```diff
@@ -314,6 +346,59 @@ def test_historical_features_from_parquet_sources(infer_event_timestamp_col):
             ).reset_index(drop=True),
         )
 
+        event_timestamp = (
+            DEFAULT_ENTITY_DF_EVENT_TIMESTAMP_COL
+            if DEFAULT_ENTITY_DF_EVENT_TIMESTAMP_COL in orders_df.columns
+            else "e_ts"
+        )
+        expected_df_fno = get_expected_training_df(
```

Comment:
lets keep the ordering of these parts the same as the above. define job, then actual_df_fno = job.to_df(), then define the expected df, then asserts. Also actual_df_fno could be named more informatively like actual_df_simple_feature_names